static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -499,3 +553,112 @@ $quote-tail: 25px;
         }
     }
 }
+// *************** Opinion Styles ***************
+
+.content--pillar-opinion {
+    .content__head {
+        overflow: hidden;
+        @include mq(phablet) {
+            overflow: visible;
+        }
+    }
+
+    .content__meta-container {
+        &:before {
+            content: none;
+        }
+    }
+
+    .content__header {
+        position: relative;
+
+        &:after {
+            @include multiline(8);
+        }
+    }
+
+    .content__headline {
+        width: 70%;
+        font-weight: 100;
+        padding-top: 0;
+        @include mq(phablet) {
+            float: left;
+        }
+    }
+
+    .content__headline--byline {
+        font-style: italic;
+        margin-left: 0;
+        padding: 0 $gs-baseline/2 $gs-gutter*3;
+        @include mq(mobileLandscape) {
+            padding-left: $gs-gutter;
+        }
+        @include mq(phablet) {
+            padding-left: 0;
+        }
+        @include mq(leftCol) {
+            padding-top: $gs-baseline;
+        }
+    }
+
+    .content__headline--byline span a {
```

Comment:
is the `span` needed here?

Reply:
yea is targeted that way, doesn't override otherwise